### DISCUSSION TITLE
"Quick compare" rows in result panels

### DISCUSSION THREAD

**Discussion by marvin-me** [dbeaver/dbeaver#35025] [Ideas]
An option to quickly compare rows in results would be useful.
Mark multiple rows => Context Menu "Compare" => Color differences in columns.

I often find myself exporting the data into Excel or Notepad to compare it.

**Comment by TB-HunBoka** [comment]
Well this sounds very usefull. I often use the Mark multiple rows -> Record view tab to compare multipre rows, and I enabled "Mark occurrences  of the selected cell's value" option. Then I move trough the values with the down arrow, and search for differences this way. 
Like this: 
![kép](https://github.com/user-attachments/assets/55c48975-fbc9-47f3-ba5c-1983f691e81a)

This functionality would help me too as I also use notepad++ or excel to compare high amount of data.
